starters/blog/src/components/seo.js

```diff
@@ -35,7 +35,7 @@ const SEO = ({ description, lang, meta, title }) => {
         lang,
       }}
       title={title}
-      titleTemplate={`%s | ${site.siteMetadata.title}`}
+      titleTemplate={`%s | ${site.siteMetadata?.title || `No title defined`}`}
```

Comment:
I'd just default to blank, or maybe something extra informaitve like "Set your site title in gatsby-config.js"